vignettes/Model.Rmd

```diff
@@ -10,90 +12,365 @@ vignette: >
+
+```{r, output.lines=6}
+head(test_sim, n = 3)
 ```
 
-This is an individual based model for P. Falciparum and malaria interventions.
+Additional output details can be found in the `run_simulation()` documentation.
+
+### Additional outputs
+
+**Age stratified** results for **incidence**, **clinical incidence** and **severe case incidence** may also be included in the output if desired and must be specified in the parameter list (see `get_parameters()` for more details and [Demography](https://mrc-ide.github.io/malariasimulation/articles/Demography.html) for an example). These inputs will add extra columns to the output for the number of infections (**`n_`**) and the sum of probabilities of infection (**`p_`**) for the relevant total, clinical or severe incidences for each specified age group.
+
+Where **treatments** are specified, `n_treated` will report the number that have received treatment. Where **bed nets** are distributed, `net_usage` specifies the number sleeping under a bednet.
 
-# Human Biology
+### Output visualisation
 
-The human variables are documented in "R/variables.R".
+These outputs can then be visualised, such as the population changes in states. Another key output is the prevalence of detectable infections between the ages of 2-10 (*Pf*PR~2-10~), which can be obtained by dividing `n_detect_730_3650` by `n_730_3650`.
 
-The human biological processes are spread out between the following files:
+```{r, fig.align = 'center', fig.height = 5, fig.width = 7}
+# Set color palette
+cols  <- c("#E69F00", "#56B4E9", "#009E73", "#F0E442", "#0072B2", "#D55E00", "#CC79A7")
 
- 1. 'human_infection.R' (orange)
- 2. 'disease_progression.R' (green)
- 3. 'mortality_processes.R'(red)
+# Define vector of column names to plot
+cols_to_plot <- paste0(c("S","D","A","U","Tr"),"_count")
 
-```{r echo=FALSE}
-DiagrammeR::grViz("digraph {
-  graph [layout = dot, rankdir = LR]
+# Create plotting function
+states_plot <- function(){
   
-  node [shape = rectangle]        
-  S
-  D
-  A
-  U
-  Tr
+  # Set up plot with first state
+  plot(x = test_sim$timestep, y = test_sim[,cols_to_plot[1]],
+       type = "l", col = cols[1], ylim = c(0,80),
+       ylab = "Population size", xlab = "Days")
   
-  node [shape = circle]
-  i [label = '']
+  # Add remaining states
+  sapply(2:5, function(x){
+    points(x = test_sim$timestep, y = test_sim[,cols_to_plot[x]], type = "l", col = cols[x])})
   
-  # edge definitions with the node IDs
-  S -> i [color='orange']
-  i -> D [color='orange']
-  i -> A [color='orange']
-  i -> Tr [color='orange']
-  D -> A -> U -> S [color='green']
-  Tr -> s [color='green']
-  A -> i [color='orange']
-  U -> i [color='orange']
-  U -> S [color='red']
-  A -> S [color='red']
-  D -> S [color='red']
-  }")
+  # Add legend
+  legend("topleft", legend = c("S","D","A","U","Tr"), col = cols, lty = 1, bty = "n", ncol = 5)
+}
+
+states_plot()
+
+# Calculate Pf PR 2-10
+test_sim$PfPR2_10 <- test_sim$n_detect_730_3650/test_sim$n_730_3650
```

Comment:
beautiful plots, could add `set_equilibrium` above to get the prevalence to be steadier over time. 
If this is leaping too far ahead in the vignettes, it is okay to leave as is, or include a note about why prevalence is increasing (model working itself into an equilbrium state)

Reply:
I've added a note - I think it's helpful to see what happens when set_equilibrium isn't used to see how much of a difference it makes.